.gitignore

```diff
@@ -24,3 +24,6 @@ api/spec/v1.json
 
 # Coverage reports.
 coverage.txt
+
+# Python build files
+tests/load/__pycache__
```

Comment:
Nit: Might as well use `**/__pycache__` -- in case of future refactors or new py projects.